src/sql/workbench/services/restore/browser/restoreDialog.ts

```diff
@@ -287,7 +287,7 @@ export class RestoreDialog extends Modal {
 			ariaLabel: LocalizedStrings.TARGETDATABASE,
 			placeholder: localize('targetDatabaseTooltip', "Please enter target database name"),
 			validationOptions: {
-				validation: (value: string) => this.viewModel.databases.includes(value) ? ({ type: MessageType.ERROR, content: localize('restoreDialog.targetDatabaseAlreadyExists', "Target database already exists") }) : null
+				validation: (value: string) => this.viewModel.databases?.includes(value) ? ({ type: MessageType.ERROR, content: localize('restoreDialog.targetDatabaseAlreadyExists', "Target database already exists") }) : null
```

Comment:
You could also just initialize the array. 

If you keep it like this please update the typing in the viewmodel to be correct, e.g. `	public databases: string[] | undefined;
`

Reply:
I changed databases field to optional similar to all the other properties in the viewmodel class.  That looks like the change that is most consistent with the current design.